ui/src/views/PageNotFoundView.vue

```diff
@@ -5,13 +5,18 @@
     tag="section"
   >
     <v-row justify="center">
-      <v-col cols="auto">
-        page not found
+      <v-col id="error-view-content">
```

Comment:
instead of coding from scratch, you can use the ContentCard component instead; that component was created so that we can standardize the look and ease the development.  XD 

Also, v-col needs `cols`

Reply:
And I think I also added slots under the `back to home`  button so that you can add the link by using that component... but please check

You can also adjust this component (ContentCard.vue) because I think right now its kinda dirty... you can make it more dynamic by adding some more props for the buttons, for example... or anything  XD